POC/ColorPicker/ColorPicker/TransparentWindow.xaml

```diff
@@ -9,6 +9,20 @@
         WindowStyle="None"
         ShowInTaskbar="False"
         Background="#01000000"
+        Loaded="OnLoad"
```

Comment:
Should this be OnLoaded instead of OnLoad?

Reply:
Yes, I will change it.